src/intrinsics.cpp

```diff
@@ -331,8 +331,7 @@ static Value *emit_unbox(Type *to, const jl_cgval_t &x, jl_value_t *jt, Value *d
 
     int alignment;
     if (x.isboxed) {
-         // julia's gc gives 16-byte aligned addresses
-        alignment = 16;
+        alignment = JL_HEAP_ALIGNMENT;
```

Comment:
This shouldn't be changed since this won't increase. It might even be too much and should be `sizeof(void*) * 2` now.